README.md

```diff
@@ -227,21 +300,43 @@ void helloWorld(ExpressHttpRequest request) async {
 }
 ```
 
-## HTTPS functions details
+## HTTPS Functions Details
 
-Firebase uses Expressjs web framework for HTTPS functions with `body-parser`
-middleware enabled by default ([documentation][body_parser_docs]).
+Firebase uses the Express.js web framework for HTTPS functions with `body-parser`
+middleware enabled by default ([documentation](https://firebase.google.com/docs/functions/http-events#read_values_from_the_request)).
```

Comment:
nit: I'd prefer URLs to always use reference-style format as oftentimes they get really long and make reading raw markdown quite a bit more painful.

Reply:
Although a bit uglier, it's a lot easier to reason about when using in line links. I'll change it back if you insist